### ISSUE TITLE
Issue with rule "Label and name from content mismatch" (2ee8b8)

### ISSUE THREAD

**Issue by annethyme** [act-rules/act-rules.github.io#493]
@mraccess77 wrote in https://github.com/act-rules/act-rules.github.io/issues/104#issuecomment-488006872: 

> I think "2" and "two" would pass as they would be voiced the same way be users of speech recognition and this is the reason for this test. Some discussion here w3c/wcag#592.

**Comment by Brynanders** [comment]
> @mraccess77 wrote in [#104 (comment)](https://github.com/act-rules/act-rules.github.io/issues/104#issuecomment-488006872):
> 
> > I think "2" and "two" would pass as they would be voiced the same way be users of speech recognition and this is the reason for this test. Some discussion here [w3c/wcag#592](https://github.com/w3c/wcag/issues/592).

Only in English though. I can't think of a case in point but passing "2" and "two" just doesn't seem right. Would you also pass "their" and "there" and "know" and "no"? Surely not.

**Comment by mraccess77** [comment]
Their and they’re would not be allowed.  I’m not sure why numbers would be different for different languages.  This rule should be harmonized with the guidance/discussion from the Accessibility Guidelines working group.

**Comment by WilcoFiers** [comment]
@audreymaniez pointed out that the understanding document of this SC specifically says not to substitute mathematical symbols for text:

> https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae
> Mathematical expressions are an exception to the previous subsection about symbolic characters. Math symbols can be used as labels; for example "11×3=33" and "A>B" convey meaning. The label should not be overwritten in the accessible name, and substitutions of words where a formula is used should be avoided since there are multiple ways to express the same equation. ...

I'll add a test case to make this explicitly part of the rule.
